packages/paste-website/next.config.mjs

```diff
@@ -28,6 +28,20 @@ const nextConfig = {
   experimental: {
     legacyBrowsers: false,
   },
+  // https://nextjs.org/docs/pages/api-reference/next-config-js/headers
```

Comment:
Note: I sneak fix in this PR so we can iframe the website from other websites. Normally this feature is defaulted off for security, because you dont want people embedding login flows into their websites and stealing credentials. However our docs doesn't have a login flow and is just information based and open source. So this is fine.